src/components/VmActions/sagas.js

```diff
@@ -4,10 +4,11 @@ import { CONSOLE_IN_USE } from './constants'
 import { setConsoleInUse } from './actions'
 
 function* getConsoleInUseModal (sagas, action) {
-  let { vmId, usbFilter } = action.payload
+  let { vmId, usbFilter, userId } = action.payload
   const sessionsInternal = yield sagas.fetchVmSessions({ vmId })
   if (sessionsInternal &&
-    sessionsInternal.find((x) => x.consoleUser) !== undefined) {
+    sessionsInternal.find((x) => x.consoleUser) !== undefined &&
```

Comment:
Can we call `sessionsInternal.find()` just once by extending the predicate?
When refactoring, please keep readability on mind.